web/impact/impact/v1/views/office_hours_calendar_view.py

```diff
@@ -282,19 +301,27 @@ def succeed(self):
         self.response_elements['header'] = self.SUCCESS_HEADER
 
 
+def _check_remote_location(locations_list):
+    remote_location_list = [
```

Comment:
Not a blocker, but you could express this as

`return any([loc['location_name']=='Remote' for loc in locations_list])`